x-pack/plugins/siem/public/components/filters_global/filters_global.tsx

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { EuiFlexGroup, EuiFlexItem } from '@elastic/eui';
+import euiDarkVars from '@elastic/eui/dist/eui_theme_dark.json';
+import euiLightVars from '@elastic/eui/dist/eui_theme_light.json';
+import React from 'react';
+import { Sticky } from 'react-sticky';
+import { pure } from 'recompose';
+import styled from 'styled-components';
+
+import { SuperDatePicker } from '../super_date_picker';
+
+const offsetChrome = 49;
+const gutterTimeline = '70px'; // Temporary until timeline is moved - MichaelMarcialis
+
+const Aside = styled.aside<{ isSticky?: boolean }>`
+  background: ${({ theme }) =>
+    theme.darkMode ? euiDarkVars.euiColorEmptyShade : euiLightVars.euiColorEmptyShade};
+  border-bottom: ${({ theme }) =>
+    theme.darkMode ? euiDarkVars.euiBorderThin : euiLightVars.euiBorderThin};
+  box-sizing: content-box;
+  margin: 0 -${gutterTimeline} 0 -${euiLightVars.euiSizeL};
+  padding: ${euiLightVars.euiSize} ${gutterTimeline} ${euiLightVars.euiSize}
+    ${euiLightVars.euiSizeL};
+
+  ${props =>
+    props.isSticky &&
+    `
+      top: ${offsetChrome}px !important;
+      z-index: ${euiLightVars.euiZNavigation};
+    `}
+`;
+
+export interface FiltersGlobalProps {
+  children?: React.ReactNode;
```

Comment:
It doesn't look like you have any usages of `FiltersGlobal` without `children` outside of your snapshot test -- does this need to be optional?

If not, you can remove the `?`, and update the component below to just use:

``` jsx
<EuiFlexItem>{children}</EuiFlexItem>
```

instead of:

``` jsx
{children && <EuiFlexItem>{children}</EuiFlexItem>}
```

And for your `filters_global.test` you could just do:
``` ts
const wrapper = shallow(<FiltersGlobal>Sample Child</FiltersGlobal>);
```

This is all if you don't intend to support an empty `FiltersGlobal` of course.

Reply:
I originally had it that way in the case that we wanted to have a page with a date picker but no KQL bar, but since that use case doesn't exist currently, I'll take out the optional :)